### PR TITLE
Connect to Iroha functionality.

### PR DESCRIPTION
**Pull request by humb1t**
Signed-off-by: Nikita Puzankov <humb1t@yandex.ru>

### CODE REVIEW

File: iroha/src/torii.rs [hunk 1]
```diff
@@ -89,6 +104,33 @@ async fn handle_connection(
...
+        stream
+            .flush()
+            .await
+            .map_err(|e| format!("Failed to flush: {}", e))?;
+        let mut receipt = [0u8; 4];
+        stream
+            .read(&mut receipt)
+            .await
+            .map_err(|e| format!("Failed to read receipt: {}", e))?;
+        if receipt != [1; 4] {
```
[the remark this answers is not in the document]
Comment (humb1t) [reply iroha/src/torii.rs:127]: https://jira.hyperledger.org/browse/IR-800 added TODO

File: iroha/src/torii.rs [hunk 2]
```diff
@@ -60,8 +66,15 @@ impl Torii {
             sumeragi_message_sender,
             block_sync_message_sender,
             system,
+            events_sender: self.events_sender.clone(),
+            events_receiver: self.events_receiver.clone(),
         };
-        Network::listen(Arc::new(RwLock::new(state)), url, handle_connection).await?;
+        let state = Arc::new(RwLock::new(state));
+        let a = Network::listen(state.clone(), "127.0.0.1:8888", handle_connections);
+        let b = Network::listen(state.clone(), url, handle_requests);
```
[the remark this answers is not in the document]
Comment (humb1t) [reply iroha/src/torii.rs:74]: https://jira.hyperledger.org/browse/IR-800

File: iroha/src/torii.rs [hunk 3]
```diff
@@ -60,8 +66,15 @@ impl Torii {
...
+        let result = a.join(b).await;
+        result.0?;
+        result.1?;
```
[the remark this answers is not in the document]
Comment (humb1t) [reply iroha/src/torii.rs:77]: Sure

File: iroha/src/torii.rs [hunk 4]
```diff
@@ -89,6 +104,33 @@ async fn handle_connection(
...
+            .flush()
+            .await
+            .map_err(|e| format!("Failed to flush: {}", e))?;
+        let mut receipt = [0u8; 4];
+        stream
+            .read(&mut receipt)
+            .await
+            .map_err(|e| format!("Failed to read receipt: {}", e))?;
+        if receipt != [1; 4] {
+            panic!("F");
```
Comment (vmarkushin) [inline iroha/src/torii.rs:128]: Probably a stub?

Comment (humb1t) [reply iroha/src/torii.rs:128]: =)

Comment (humb1t) [reply iroha/src/torii.rs:128]: https://jira.hyperledger.org/browse/IR-800

**Pull request merged**